src/components/Project/ImageAnalysis/ImageDownloadMenu.vue

```diff
@@ -0,0 +1,64 @@
+<script setup>
+import { onMounted } from 'vue'
+import { useAlertsStore } from '@/stores/alerts'
+
+const props = defineProps(['image', 'tifUrl'])
+
+const emit = defineEmits(['analysisAction'])
+
+const alertStore = useAlertsStore()
+
+function downloadLink(link, filename, fileType) {
+  if(!link){
+    alertStore.setAlert('error', `No ${fileType} available for download`)
+    return
+  }
+  const a = document.createElement('a')
+  a.href = link
+  a.download = filename
+  document.body.appendChild(a)
+  a.click()
+  document.body.removeChild(a)
+}
+
+onMounted(() => {
+  const getTifInput = {
+    'basename': props.image.basename
+  }
+  emit('analysisAction', 'get-tif', getTifInput)
+})
+
+</script>
+<template>
+  <v-speed-dial
+    location="left center"
+    transition="fade-transition"
```

Comment:
I like the fade transition, but I wonder if it might make more sense having the menu go down from the button rather than to the left of the button?

Reply:
I chose the left position to keep the input from overlapping info below it and avoid finger blockage on tablets. When tapped, options are easily visible without needing to lift your finger. While placing it below is more common, I think the advantages of the left position are worth it